ci-operator/config/openshift/cluster-api-provider-openstack/openshift-cluster-api-provider-openstack-main.yaml

```diff
@@ -39,19 +39,30 @@ resources:
       memory: 200Mi
 tests:
 - as: verify
-  commands: make XDG_DATA_HOME=/tmp/data XDG_CACHE_HOME=/tmp/cache verify-modules
-    verify-gen verify-vendoring
+  commands: make XDG_DATA_HOME=/tmp/data XDG_CACHE_HOME=/tmp/cache verify-modules verify-gen verify-vendoring
   container:
     from: src
 - as: test
   commands: make KUBEBUILDER_ASSETS_DIR=/tmp/data test
   container:
     from: src
 - as: test-openshift
-  commands: make -C openshift XDG_DATA_HOME=/tmp/data XDG_CACHE_HOME=/tmp/cache KUBEBUILDER_ASSETS_DIR=/tmp/data
-    verify test
+  commands: make -C openshift XDG_DATA_HOME=/tmp/data XDG_CACHE_HOME=/tmp/cache KUBEBUILDER_ASSETS_DIR=/tmp/data verify test
   container:
     from: src
+- as: e2e
+  steps:
+    cluster_profile: openstack-vexxhost
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
+    test:
+    - as: test
+      commands: make -C openshift e2e
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+    workflow: openshift-e2e-openstack-techpreview
```

Comment:
Do not use this deprecated workflow. Please use openshift-e2e-openstack-ipi, which should already set the feature set you set with `FEATURE_SET`.

Reply:
Done